.evergreen/.evg.yml

```diff
@@ -533,16 +533,20 @@ buildvariants:
 #  display_name: "${version} ${topology} ${auth-and-ssl} ${ruby} ${os}"
 #  tasks:
 #     - name: "test"
-- matrix_name: "rhel70-all-mongodbs-all-rubies-all-topologies-all-auth-and-ssl"
+-
+  #matrix_name: "rhel70-all-mongodbs-all-rubies-all-topologies-all-auth-and-ssl"
```

Comment:
We can delete this instead of commenting; if we want to go back at some point, we can just check the git history.